pipeline_steps/pubcloud.groovy

```diff
@@ -227,7 +227,10 @@ def runonpubcloud(Map args=[:], Closure body){
       } catch (e){
         print "Error while cleaning up, swallowing this exception to prevent "\
               +"cleanup errors from failing the build: ${e}"
-        common.create_jira_issue("RE", "Cleanup Failure: ${env.BUILD_TAG}")
+        common.create_jira_issue("RE",
+                                 "Cleanup Failure: ${env.BUILD_TAG}",
+                                 "[${env.BUILD_TAG}|${env.BUILD_URL}]",
+                                 ["jenkins", "cleanup_fail"])
```

Comment:
Do we want to set a standard for RE-specific jobs where we choose either create or create/append?  Is it purely down to the job or do we want to always choose one over the other?

Reply:
I think cleanup failures are different from build failures as they are always an RE problem, not a project problem. We prefer a card per failure, so that's what we'll get.